guides/Step By Step Guide/CustomizingTheClient.md

```diff
@@ -0,0 +1,98 @@
+Congrats! If you reached this part of the guide, you have successfully created your very own Discord Bot! Now, its time to understand the options Klasa gives us so we can customize the bot.
+
+> Note: This guide was intended for beginner developers so we will go step by step. However, if you wish to speed things up you can use this code snippet below so you don't have to keep restarting the bot. Otherwise, just follow along with the guide to understand how each of these options work.
+
+```js
+const BotClient = new Client({
+	commandEditing: true,
+	commandLogging: true,
+	prefix: `+`,
+	readyMessage: (client) => `${client.user.tag}, ready to serve ${client.guilds.size} guilds.`
+});
+```
+
+## Adding Your Own Default Prefix
+
+Remember how we used `@bot help` to get the bot to send us the help command. The reason `@bot` worked is because Klasa adds the bots @ as a default prefix.
```

Comment:
```suggestion
Remember how we used `@bot help` to get the bot to send us the help command. The reason `@bot` worked is because Klasa adds the bots mention tag as a default prefix.
```